code/src/WijmoProvider/Features/ColumnFilter.ts

```diff
@@ -108,6 +108,62 @@ namespace WijmoProvider.Feature {
             this.setState(this._enabled);
         }
 
+        public byCondition(
+            columnId: string,
+            values: OSFramework.OSStructure.FilterCondition[]
+        ): void {
+            const column = this._grid.getColumn(columnId);
+            if (column) {
+                const columnFilter = this._filter.getColumnFilter(
+                    column.config.binding
+                ).conditionFilter;
+
+                if (values.length > 0) {
+                    const condition1 = values[0];
+                    const condition2 = values[1];
+
+                    columnFilter.condition1.value =
+                        column.columnType === OSFramework.Enum.ColumnType.Number
+                            ? parseInt(condition1.value)
```

Comment:
If column type is Number, the code is doing parseInt, shouldn't be for parseFloat? What about for Currency, Date and DateTime?

Reply:
You're right! Changed to parseFloat. Currency, date and datetime column expect a string.